src/Templating/Helper/PhoneNumberHelper.php

```diff
@@ -67,22 +70,26 @@ public function format(PhoneNumber $phoneNumber, $format = PhoneNumberFormat::IN
     /**
      * Formats this phone number for out-of-country dialing purposes.
      *
-     * @param PhoneNumber $phoneNumber phone number
+     * @param PhoneNumber|string $phoneNumber phone number
      * @param string|null $regionCode  The ISO 3166-1 alpha-2 country code
      */
-    public function formatOutOfCountryCallingNumber(PhoneNumber $phoneNumber, $regionCode): string
+    public function formatOutOfCountryCallingNumber($phoneNumber, $regionCode): string
```

Comment:
```suggestion
    public function formatOutOfCountryCallingNumber(PhoneNumber|string $phoneNumber, $regionCode): string
```

This is a nice reason to bump minimum PHP version to 8.0 WDYT?

Reply:
I'd be glad, is there any policy on how to do that? It will be a major bump? Or minor?

Reply:
We can bump dependencies on minor versions. However, covering this upgrade to PHP 8.0 should be done in another PR (it means several changes).

Reply:
We can but it must be done in another RP because there are other typing that could be included

Reply:
Let's merge it, and I can prepare separate PR to bump version 👍

Reply:
as promised https://github.com/odolbeau/phone-number-bundle/pull/143